1.1/spec/08-Part-05.adoc

```diff
@@ -47,5 +47,5 @@ The URI `ogc:asGeomLiteral` is used in requirement specifications as a placehold
 
 ==== RDF Serializations
 
-Two RDF serializations are used in this document. Terse RDF Triple Language (turtle) [10] is used for RDF snippets placed within the main body of the document, and RDF/XML [11] is used for the examples in Annex B.
+Two RDF serializations are used in this document. Terse RDF Triple Language (turtle) <<TURTLE>> is used for RDF snippets placed within the main body of the document, and RDF/XML <<RDFXML>> is used for the examples in Annex B.
```

Comment:
RDF/XML should die!
No-one seriously working with RDF or ontologies uses RDF/XML seriously any more. 
Turtle and JSON-LD only

Reply:
Not in scope for this PR but I am addressing this - turtle in the doc, turtle + JSON-LD for the examples, in my larger update PR to be lodged after these small ones are done.